src/app/containers/InlineLink/index.test.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { StaticRouter } from 'react-router-dom';
+import { shouldMatchSnapshot } from '../../helpers/tests/testHelpers';
+import InlineLink from './index';
+
+describe('InlineLink', () => {
+  describe('internal link', () => {
```

Comment:
Renaming these descriptions would add clarity. 
In most cases we use "internal link" to mean "a link going to another BBC page" but here you're referring to "a link within this SPA". Same for "external link" below.